data/172/939/588/5/1729395885.geojson

```diff
@@ -34,10 +40,11 @@
     ],
     "wof:breaches":[],
     "wof:concordances":{
-        "uscensus:geoid":2000359175
+        "uscensus:geoid":"2000359175"
```

Comment:
Adding quoting for "uscensus:geoid" values is deliberate as the occasional leading `0` (like `06` prefix for places in **California**) is significant and the whole value should be treated as a string.

Reply:
Makes sense